### ISSUE TITLE
Add support for 300S Air Purifier PlasmaPro version (LAP-C302S-WUSB)

### ISSUE THREAD

**Issue by vlebourl** [webdjoe/pyvesync#179]

**Comment by vlebourl** [comment]
Hi, 

Following up on an issue on my Home Assistant custom comp relying on your library.
https://github.com/vlebourl/custom_vesync/issues/114

Any chance you could add this to your lib? 

Cheers.

**Comment by vlebourl** [comment]
duplicate of #152

**Comment by vlebourl** [comment]
waiting on https://github.com/webdjoe/pyvesync/pull/160
